pkg/status/render/templates/endpoints.tmpl

```diff
@@ -1,14 +1,15 @@
 {{/*
 NOTE: Changes made to this template should be reflected on the following templates, if applicable:
 * cmd/agent/gui/views/templates/generalStatus.tmpl
-* Dockerfiles/cluster-agent/dist/templates/forwarder.tmpl
```

Comment:
Is this no longer needed ?

Reply:
I build the cluster agent using `inv cluster-agent.build`

Reply:
indeed !